_docs/sql-reference/sql-commands/021-create-schema.md

```diff
@@ -446,7 +467,8 @@ Note that date, time type conversion uses the Joda time library, thus the format
 
 
 ## Limitations
-This feature is currently in the alpha phase (preview, experimental) for Drill 1.16 and only applies to text (CSV) files in this release. You must enable this feature through the `exec.storage.enable_v3_text_reader` and `store.table.use_schema_file` system/session options.
+This feature applies to format plugins that use the `Enhanced Vector Framework`. You must enable this feature through
```

Comment:
((Only developers know about EVF. Maybe:))

Schema provisioning works with selected readers. If you develop a format plugin, you must use the `Enhanced Vector Framework` (rather than the "classic" techniques) to enable schema support.

To use schema provisioning, you must first enable it with the ... option.

Reply:
Thanks, replaced.